src/helpers/getDisplayPrice.js

```diff
@@ -1,16 +1,6 @@
 import isEqual from 'lodash.isequal'
 
-const sortNumber = (a, b) => {
-  if (a > b) return 1
-  if (a < b) return -1
-  return 0
-}
-
-// DEMO -> https://jsperf.com/es6-array-unique/1
-const uniqueFilter = entries => entries.filter((elem, pos, arr) => arr.indexOf(elem) === pos)
-
-const isValidType = value =>
-  (typeof value === 'string' || typeof value === 'number') && (value || value >= 0)
+const CURRENCY_SYMBOL = '€'
 
 const valueToNumber = v => {
```

Comment:
```suggestion
const valueToNumber = value => {
```
Pour être cohérent avec `valueToPrice`.

Reply:
ok